edge/pkg/metamanager/metaserver/common/types.go

```diff
@@ -4,3 +4,27 @@ type RestartInfo struct {
 	Namespace string
 	PodNames  []string
 }
+
+type LogsInfo struct {
+	Namespace                    string `query:"-"`
+	PodName                      string `query:"-"`
+	ContainerName                string `query:"-"`
+	TailLines                    string `query:"tailLines"`
+	Follow                       string `query:"follow"`
+	InsecureSkipTLSVerifyBackend string `query:"insecureSkipTLSVerifyBackend"`
+	LimitBytes                   string `query:"limitBytes"`
+	Pretty                       string `query:"pretty"`
+	SinceSeconds                 string `query:"sinceSeconds"`
+	Timestamps                   string `query:"timestamps"`
+}
+
+type ExecInfo struct {
```

Comment:
Why param. in ExecInfo don't need the format `query:"xxx"` like LogsInfo?

Reply:
The `logs` is run on the edged restful API, these tags is used to generate query parameters. `exec` calls the `remoteRuntimeService` methods (`ExecSync` and `Exec`) without generating query parameters.